### PR TITLE
🛠️ Made code simpler, customizable and removed unnecessary stuff

### PR DESCRIPTION
**Pull request by fsocietystudios**
🛠️ Updated ``cutscene.lua``
- Made code simpler, and easier to read.
- Removed ``IsMale`` function, there is a native function we can use.
- Added customization support for the passengers.
- Added ESX support for auto triggering the event when a player joins the server.
- Added comments (Most of them are new)

🛠️ Added config file
- Added configuration file.

🛠️ Updated resource manifest
- Added the used files to the resource manifest.

🛠️ TODO:
- For now, its adding random textures to all of the passengers. (Generating random texture id from the range of the options). For a future update, textures will be customizable too.

To sum up, I have learned a lot from that.
I'll be more then happy if you will accept this PR! (Actually my first ;D)
Thanks! ❤️

### CODE REVIEW

**Review by Doublox — APPROVED** [review]

**Review by Doublox — APPROVED** [review]

**Comment by Doublox** [comment]
🗡️ :)
